packages/@glimmer/application/src/components/component-managers/template-only.ts

```diff
@@ -28,24 +27,23 @@ import {
 } from '../component-definition';
 
 export const CAPABILITIES: ComponentCapabilities = {
-  attributeHook: true,
-  createArgs: true,
+  attributeHook: false,
+  createArgs: false,
   createCaller: false,
   createInstance: true,
   dynamicLayout: false,
   dynamicScope: false,
-  dynamicTag: true,
-  elementHook: true,
+  dynamicTag: false,
+  elementHook: false,
   prepareArgs: false,
-  updateHook: true,
+  updateHook: false,
```

Comment:
FWIW, in Ember this (and `createArgs`) are conditionally set to `true` when the render debug support is enabled. See 

https://github.com/emberjs/ember.js/blob/016f8066d0b484f3f69ce8c337be3b8b8aa5fce4/packages/%40ember/-internals/glimmer/lib/component-managers/template-only.ts#L18-L29

This is specifically, so that we can properly update the render tree in the Ember inspector. I do not think this specific code is going to be used when `@glimmer/component` is used in an Ember app, but I wanted to point out this difference just in case.